pdc/apps/release/serializers.py

```diff
@@ -205,3 +207,34 @@ class VariantTypeSerializer(StrictSerializerMixin, serializers.ModelSerializer):
     class Meta:
         model = VariantType
         fields = ('name',)
+
+
+class ReleaseGroupSerializer(StrictSerializerMixin, serializers.ModelSerializer):
+    description         = serializers.CharField(required=True)
+    type                = serializers.SlugRelatedField(slug_field='name',
+                                                       queryset=ReleaseGroupType.objects.all())
+    releases            = ChoiceSlugField(slug_field='release_id',
+                                          many=True, queryset=Release.objects.all(),
+                                          allow_null=True, required=False)
+    active              = serializers.BooleanField(default=True)
+
+    class Meta:
+        model = ReleaseGroup
+        fields = ('name', 'description', 'type', 'releases', 'active')
+
+    def to_internal_value(self, data):
+        releases = data.get('releases', None)
```

Comment:
If the fallback was `[]` instead of `None`, the `if` condition would no longer be necessary.